File: Tema 3 -Sentencia condicional (if-switch)/ejercicios/ejercicio14 si un numero es par y divisible/archives/14.py
import sys
import math

# Obtener los argumentos de la línea de comandos
a = int(sys.argv[1])
par = "Es par" if a % 2 == 0 else "No es par"
div = "y es divisible entre 5" if a % 5 == 0 else "y no es divisible entre 5"
mensaje = f"El numero {a}: <br>{par}<br>{div}"
print(f"{mensaje}")

File: Tema 3 -Sentencia condicional (if-switch)/ejercicios/ejercicio14 si un numero es par y divisible/archives/14.js
function sumarJS() {
  const a = parseInt(document.getElementById("num1").value);
  if (Number.isFinite(a)) {
    /* SECTION inicio codigo tras verificacion de formulario */
    mensaje = `El numero ${a}: 
    <br>${a % 2 == 0 ? "Es par" : "No es par"}
    <br>${a % 5 == 0 ? "y es divisible entre 5" : "y no es divisible entre 5"}`;
    /* !SECTION fin codigo tras verificacion de formulario */
  } else {
    mensaje = `Ingrese un número válido en el formulario `;
  }
  const resultadoDiv = document.getElementById("resultadoJavascript");
  resultadoDiv.innerHTML = `${mensaje}`;
}
